Classes/Queue/RabbitQueue.php

```diff
@@ -133,9 +148,16 @@ public function __construct(string $name, array $options = [])
             if ($this->exchangeName !== '') {
                 $this->channel->queue_bind($this->queueName, $this->exchangeName, $this->routingKey);
             }
-        }
 
-        $this->useDLX = $options['useDLX'] ?? false;
+            // bind the queue to the dlx
+            if ($this->useDLX && isset($queueOptions['arguments']['x-dead-letter-routing-key'])) {
```

Comment:
I think it should be explicitly configurable whether a new binding is added or not